packages/EasyErrorHandler/src/Bridge/Symfony/Commands/AnalyzeErrorCodesCommand.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Symfony\Commands;
+
+use EonX\EasyErrorHandler\Bridge\Symfony\Interfaces\ErrorCodes\ErrorCodesProviderInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class AnalyzeErrorCodesCommand extends Command
+{
+    /**
+     * @var string
+     */
+    protected static $defaultName = 'easy-error-handler:error-codes:analyze';
+
+    public function __construct(
+        private ErrorCodesProviderInterface $errorCodesProvider,
+        private int $categorySize = 100
```

Comment:
Sorry but I think we shouldn't use a default value here because of company's code style rules.
I should be `private ?int $categorySize = null` and then in the body `$this->categorySize = $categorySize ?? 100;`.

Reply:
In new projects, for such fields that have a default value, we define them in this form:
```
protected string $status = self::STATUS_ACTIVE
...
protected int $priority = 0
```

Reply:
As I know it only related to entities and properties with the `readonly` modifier.
@akl-95 wdyt?

Reply:
Yes, this discussion initially appeared when working on `readonly` PHP feature. But here we have a little different case.
Let's consider what structure would we have if we don't use promoted properties:
```PHP
    private int $categorySize;

    private ErrorCodesProviderInterface $errorCodesProvider;

    public function __construct(
        ErrorCodesProviderInterface $errorCodesProvider,
        ?int $categorySize = null
    ) {
        $this->categorySize = $categorySize ?? 100;
        parent::__construct();
    }
```

That means we have to follow one of the ways:
 - don't use promoted properties at all
 - use non-null default value for the promoted property
 
 Otherwise, we'll have the nullable property that actually is not nullable.
 So it looks pretty similar to the readonly-related issue. If we want to use promoted property here (and in other similar places) we can't use default null value, and we should update our rules.

Reply:
I just realised that I didn't use the promoted properties for such parameters in my [PR](https://github.com/eonx-com/easy-monorepo/pull/897/files#diff-19585bd0ab2a39fbfa3b4359912c3e198156561e14b4d9c4c630eb03a3c81058R34).

Reply:
I'm ok with changing the standard yeah, to use non default null value so we can use promoted properties